### ISSUE TITLE
Application page: doesn't explain what the apps are

### ISSUE THREAD

**Issue by andreasn** [flatpak/flatpak.github.io#155]
I was showing Fabiana the Flatpak website yesterday, since we used the laptop with Fedora Atomic Workstation on it to look up a thing, and then had to install Telegram to get a link that she sent from another device. Good feedback overall about the website from her.

Anyway, she found the Apps section pretty neat, and recognized a bunch of the apps, but didn't quite know what "Peek", "GNOME MPV" and "Picard" was. Neither did I.
We had to look those up. She had actually been looking for something like Peek for work.

Maybe we could link to the project websites from the app page to solve that issue.

**Comment by TingPing** [comment]
Looks like there is enough empty space for a simple description too. "GNOME MPV - Video Player"

**Comment by andreasn** [comment]
yeah, that could work too!

**Comment by prcutler** [comment]
Are you thinking something like this?  Or is it too wordy?

![flatpak-website](https://cloud.githubusercontent.com/assets/67276/24879130/6bbb194a-1dfb-11e7-9b5b-d7dc7bdb1ce5.png)

**Comment by allanday** [comment]
My original mockup for that page: https://www.dropbox.com/s/jka4b92gzgkzp6l/apps-page.png?dl=0

**Comment by andreasn** [comment]
> Are you thinking something like this? Or is it too wordy?

It feels great to work on bugs with you again, Paul!
Libreoffice and Spotify are good lenghts, but the Blender is quite long. I think shorter is better, allowing it to fit on one line, like in the mockup by @allanday

**Comment by jimmac** [comment]
Altered the layout and the helper function to accomodate descriptions and bring the layout closer to the original wireframe. Please provide more fitting short descriptions.

**Comment by prcutler** [comment]
I'll work on shorter descriptions and finishing the rest and send a PR later today.

**Comment by prcutler** [comment]
jimmac: Thanks for the update to the layout and starting the short descriptions.  

I've created a pull request to finish the short descriptions: https://github.com/flatpak/flatpak.github.io/pull/156

**Comment by allanday** [comment]
I would personally lose the full stops after the descriptions, as this makes them sound rather abrupt.

Apps already provide a short, generic description in their desktop files (in the GenericName field). It might be a good idea to use these rather than making up our own strings; they're already displayed in GNOME Software.

**Comment by prcutler** [comment]
I can do a PR later today to remove the periods.

As far as the GenericName field - some are better than others, so I don't know if we want to use them.  For example, a few that I don't think are great:

FeedReader: RSS client for various webservices

Polari: An Internet Relay Chat Client for GNOME (which is redundant for our needs)

Recipes: GNOME loves to cook

Others, like LibreOffice, would still require us to write our own, as Software breaks it into its core apps.  For the closed source apps, I'll have to install them and see what they show in Software.  

With that said, I'm not sure what the best solution is.

**Comment by allanday** [comment]
> As far as the GenericName field - some are better than others, so I don't know if we want to use them. For example, a few that I don't think are great:

Heh, fair enough. The ones I looked at seemed OK, but those examples don't seem fantastic indeed. (Unelss this could be an opportunity to propose fixes to the bad generic names?!)
